readme.md

```diff
@@ -50,14 +50,14 @@ partial(2); //=> 42
 
 ## Currying
 
-> The process of converting a function with multiple arity into the same function with an arity of one. Not to be confused with partial application, which can produce a function with an arity greater than one.
+> The process of converting a function of multiple parameters into one, which, when called with fewer parameters, returns a new function that awaits the remaining parameters.  It's important to note that this returned function, if there are still more than one parameter unspecified, is also curried.  Strictly speaking, it is only currying when the returned function accepts just one parameter, but many implementations allow for multiple parameters at each level.  This is not to be confused with partial application, in which the returned function still requires all remaining parameters to be supplied.
```

Comment:
> If you partially apply the first parameter to `f`, you get a plain binary function.

Does it make more sense to discuss partial application in the context of curried functions? Given

```javascript
// f :: *,*,* -> *
const f = (a, b, c) => …;
```

and

```javascript
// g :: * -> * -> * -> *
const g = curry(f);
```

we could simply use

```javascript
g(x)  // :: * -> * -> *
```

as an example of partial application rather than

```javascript
partial(f, x)  // :: *,* -> *
```

which would need an accompanying explanation of `partial`.

We could then mention that it is possible to partially apply a function which expects multiple arguments at once, but that this requires the use of a function written specifically for this purpose.